content/en/account_management/authen_mapping/_index.md

```diff
@@ -0,0 +1,487 @@
+---
+title: Federated Authentication to Role Mapping API
+kind: documentation
+beta: true
+further_reading:
+- link: "account_management/rbac/log_management/"
+  tag: "Documentation"
+  text: "RBAC for Log Management"
+---
+
+<div class="alert alert-warning">
+Ask your customer success manager or <a href="/help">Datadog Support</a> to enable this feature.
+</div>
+
+If you are using Federated Authentication mechanisms, this API allows you to automatically map groups of users to roles in Datadog.
+
+**Note**: If you are a SAML user and you have been using the existing beta Federated Mapping mechanism (roles_v2_saml), you should transition to using this API.
+
+## Requests
+
+All the API endpoints below can have two different host endpoints:
+
+* If you are on the Datadog US site: `https://api.datadoghq.com/api/`
+* If you are on the Datadog EU site: `https://api.datadoghq.eu/api/`
+
+### Create a new Authentication mapping
+
+
+Create a new AuthN Mapping from json body. Returns the newly created AuthN Mapping.
+
+| Method | Endpoint path | Required payload |
+|--------|--------------|------------------|
+| `POST`  | `/v2/authn_mappings`  | JSON       |
+
+##### ARGUMENTS
+
+* **`role_uuid`** [*required*, *default*=none]:
+  The Roles API can be used to create and manage Datadog roles, what global permissions they grant, and which users belong to them. When you create a Role, it is assigned a UUID. For more information about finding your `role_uuid`, see the [Role API documentation][1].
+* **`attribute_key`** [*required*, *default*=none]:
+ The `attribute_key` and the `attribute_value` are key/value pairs defined in their SAML assertions from their Identity Providers. You can define these for your own use case. For example, `attribute_key` could be `member-of` and the `attribute_value` could be `Development`.
+* **`attribute_value`** [*required*, *default*=none]:
+  The `attribute_key` and the `attribute_value` are key/value pairs defined in their SAML assertions from their Identity Providers. You can define these for your own use case. For example, `attribute_key` could be `member-of` and the `attribute_value` could be `Development`.
+
+{{< tabs >}}
+{{% tab "Example" %}}
+
+{{< code-block lang="bash" filename="roles.sh" >}}
+{�
+  "type": "authnmapping",
+  "data": {�
+    "attributes": {�
+      "role_uuid": "00000000-0000-0000-0000-000000000000",
+      "attribute_key": "string",
+      "attribute_key": "string"
+    }
+  }
+}
+{{< /code-block >}}
+
+Replace the `<YOUR_DATADOG_API_KEY>` and `<YOUR_DATADOG_APPLICATION_KEY>` placeholder with the corresponding [API and application keys for your organization][1].
+
+[1]: https://app.datadoghq.com/account/settings#api
+{{% /tab %}}
+{{% tab "Response" %}}
+
+{{< code-block lang="json" filename="response.json" disable_copy="true" >}}
+{
+  "data": {
+    "attributes": {
+      "created_at": "string",
+      "created_by": "string",
+      "uuid": "10000000-abcd-efgh-ijkl-000000000000",
```

Comment:
As per the most recent version of the authn_mapping_spec, there is no more `uuid` field, as it is double counted by the `id` field